src/pages/my-bookings-page/my-bookings-page.tsx

```diff
@@ -0,0 +1,88 @@
+import classNames from 'classnames';
+import styles from './my-bookings-page.module.scss';
+import { useBookingHistory, useUpcomingBookings, useWixApi } from '../../api/api-hooks';
+import commonStyles from '../../styles/common-styles.module.scss';
+import { TabItem, Tabs } from '/src/components/tabs/tabs';
+import { format, getDate } from 'date-fns';
+import { extendedBookings } from '@wix/bookings';
+
+export type MyBookingsPageProps = {
+    className?: string;
+}
+
+export type BookingProps = {
+    booking: extendedBookings.Booking;
+    onCancel?: (booking: extendedBookings.Booking) => void;
+}
+
+enum SelectedView {
+    UPCOMING = 'UPCOMING',
+    HISTORY = 'HISTORY',
+}
+
+
+
+const Booking = ({ booking, onCancel }: BookingProps) => {
+    const date = booking.startDate!;
+    return (
+        <div className={styles['booking-root']}>
+            <div className={styles.content}>
+                <div className={styles.date}>
+                    <h3 className={styles.day}>{getDate(date)}</h3>
+                    <div className={styles['date-short']}>
+                        <span className={styles.dayInfo}>{format(date, 'EEE')}</span>
+                        <span className={styles['date-month-short']}>{format(date, 'MMM')}</span>
+                    </div>
+                </div>
+                <div className={styles.description}>
+                    <span className={styles.label}>{booking.bookedEntity!.title}</span>
+                    <span className={styles['full-date']}>
+                        {format(date, `MMM d '·' EEE, h:mm a`)}
+                    </span>
+                </div>
+            </div>
+            {onCancel && (
+                <button className={classNames(commonStyles.primaryButton, styles['cancel-button'])} onClick={() => onCancel(booking)}>
```

Comment:
`cancel-button` isn't declared, also we might want to use `secondaryButton` instead of `primaryButton` for cancel.